tests/tools.vitruv.applications.pcmjava.tests/src/tools/vitruv/applications/pcmjava/tests/pcm2java/repository/BasicComponentMappingTransformationTestNew.xtend

```diff
@@ -0,0 +1,83 @@
+package tools.vitruv.applications.pcmjava.tests.pcm2java.repository;
+
+import java.util.List
+import org.junit.jupiter.api.Test
+import tools.vitruv.applications.pcmjava.tests.pcm2java.Pcm2JavaTestUtils
+import tools.vitruv.applications.pcmjava.tests.pcm2java.Pcm2JavaTransformationTest
+import tools.vitruv.applications.pcmjava.tests.pcm2java.javahelper.JavaClassBuilder
+import tools.vitruv.applications.pcmjava.tests.pcm2java.javahelper.JavaTypeHelper
+
+import static tools.vitruv.applications.pcmjava.tests.pcm2java.PcmCreatorsUtil.*
+import static tools.vitruv.applications.pcmjava.tests.pcm2java.PcmQueryUtil.*
+
+class BasicComponentMappingTransformationTestNew extends Pcm2JavaTransformationTest {
```

Comment:
Please do not add suffixes like `New` if you rewrite classes, rather rename the old one by adding something like `legacy` or remove them.